---
layout: page
title: Course Information
permalink: /about/
menubar_toc: true
nav_order : 2
hide_hero: True
---

## General course objectives

Students should become familiar with, and enable them to use bioinformatics related to immunology and vaccine design.

## Learning objectives
A student who has met the objectives of the course will be able to:

- Explain the difference between class I and class II MHC proteins.
- Explain the structure and function of an antibody/B cell receptor.
- Find relevant immunological databases on the WWW and extract desired data.
- Identify the used germ-line genes in a final rearrangement of antibody encoding genes.
- Use web-based tools for the analysis of repertoires of immunological molecules
- Predict the possible binding of a polypeptide to the TAP transporter, and cleavage by the proteasome and describe the background for the prediction.
- Predict the possible binding of a peptide to the MHC and describe the background for the prediction.
- Predict the possible T cell epitopes in a protein sequence and describe the background for the prediction.
- Predict the possible linear and conformational B cell epitopes in a protein sequence and describe the background for the prediction.
- Use web based methods to assembe genomes and predict proteomes from next generation sequence data and describe the background for this.
- Construct a phylogenetic tree from related nucleotide sequences using the PAUP program, and identify positively selected sites by likelihood ratio testing on a suitable set of alternative models, as implemented in the program PAML.

## Content

Introduction to immunological bioinformatics, the immune system, cellular immunity, antibody mediated immunity, infections, vaccine design, modeling the immune system in the genome era, information theory of the immune system, databases and web resources in immunological bioinformatics, systems biology, experimental and theoretical description of peptide-MHC binding, prediction of epitopes using neural networks and sequence analysis, selection of epitopes using bioinformatics, prediction of proteasome processing, and TAP binding, DNA vaccines and maturation of the immune response, plasmid design, discovery of disease related genes using analysis of DNA array data, modeling of structural epitopes, intellectual property rights in bioinformatics, practical exercises in alignment, database search and prediction.

The first part will be lectures in the morning and practical exercises in the afternoon. Second part will be project work combined with lectures of invited speakers.
The evaluated exercises/projects will be done in groups.

Exercises for each of the 7 lectures with exercises and a larger group project to be presented as a poster or by power point presentation.

